mypy-README.md

```diff
@@ -11,3 +12,53 @@ mypy
 ```
 
 Note that the minimum version of MyPy that is supported is 0.770
+
+
+## What we're aiming for
+
+1. Complete type annotations for the whole code base, and shipping those in a
+   PEP 561 compatible manner (adding a `py.typed` file so the installed package
+   supports typechecking.
+2. Inline type annotations for all Python code, _except_ if there are too many
+   overloads for functions/methods - in that case a stub file should be
+   preferred (what's too many is a bit of a judgement call, I'd suggest two per
+   function is a reasonable threshold).
+3. Stub files for the extension modules (e.g. `torch._C`).
+4. Good type annotation test coverage, by using `check_untyped_defs=True` for
+   the test suite (or adding type annotations to tests).
```

Comment:
Given the brevity, it may be beneficial to propose a target for the `[mypy]` section of `mypy.ini` to further clarify the vision.